guides/source/action_cable_overview.md

```diff
@@ -651,28 +651,28 @@ consumer.subscriptions.create("AppearanceChannel", {
 #### Client-Server Interaction
 
 1. **Client** connects to the **Server** via `createConsumer()`. (`consumer.js`). The
-**Server** identifies this connection by `current_user`.
+  **Server** identifies this connection by `current_user`.
```

Comment:
There is more context in 0be7ee5, but essentially there are two cases we handle [MD032](https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md032---lists-should-be-surrounded-by-blank-lines).

In this case, I opted for indentation, but could alternatively unwrap this list item as well.

I still want this rule to prevent things like mashed up lists and other content:

```markdown
* a list item
* second item
some other content
* another list
* etc
## maybe a header
```

Will fail without proper line-breaks between the elements.